### PR TITLE
Unittesting added for bridge_common

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
unit testing added for bridge common and contents of __init__ file is moved to a new file etcdobj to solve  log problem

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Tendrl/bridge_common/pull/20" target="_blank">ReviewNinja

File: bridge_common/tests/base.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
-from oslotest import base
+import unittest
```
Comment (r0h4n) [inline bridge_common/tests/base.py]: We are using py.test which doesnt require you to inherit "unittest.TestCase"

please read more http://doc.pytest.org/

Comment (GowthamShanmugam) [inline bridge_common/tests/base.py]: i will change it

**Comment by r0h4n** [comment]
Seems like there's py.test related dependency issues, Check travis for more details and send out a new fix please

**Pull request closed without merging**